### ISSUE TITLE
Can't see job types

### ISSUE THREAD

**Issue by eXtremeX** [bee-queue/arena#39]
Hello, I tried following instructions but I can't see job types, I can however see everything else.
Would be really thankful if someone could point out what I'm doing wrong.

```
const queues = [{
    "name": "ticketVerification",
    "port": 6379,
    "host": "127.0.0.1",
    "hostId": "main"
  }]

const arena = Arena({
  queues: queues
});

const verificationQueue = new Queue('ticketVerification', 'redis://127.0.0.1:6379');

app.use('/', arena);
```

If I type in redis-cli keys *, I can see

```
127.0.0.1:6379> keys *
1) "bull:ticketVerification:1"
2) "bull:ticketVerification:id"
3) "bull:ticketVerification:failed"
```
I've attached screenshot.

Thank you very much for trying to help!
![problem](https://user-images.githubusercontent.com/8009142/29235601-68428fd0-7f00-11e7-9888-bea108c3cfae.png)

**Comment by dreamcatcha** [comment]
Same issue here, I thought it might have something to do with the prefix param but defaults to bull when not supplied, and I have tried both ways.

**Comment by eXtremeX** [comment]
@dreamcatcha I, too, tried it both ways. Either we're doing something silly, or something's wrong. If it is the former, we could improve readme a bit after we figure it out!

**Comment by dreamcatcha** [comment]
Absolutely ;)

**Comment by ccw** [comment]
It should be caused because the usage of `getJobCounts` function [here](https://github.com/bee-queue/arena/blob/master/src/server/views/dashboard/queueDetails.js#L15). According to [bull's API reference](https://github.com/OptimalBits/bull/blob/master/REFERENCE.md#queuegetjobcounts), it should be called without giving any parameters.

**Comment by ccw** [comment]
Same issue [here](https://github.com/bee-queue/arena/blob/master/src/server/views/dashboard/queueJobsByState.js#L24) causes the paging not working.

**Comment by leemason** [comment]
can this pull request be merged? i too have the same issue and cant resolve.

**Comment by bradvogel** [comment]
Yes, sorry for the delay. Published as 2.1.2
